crates/uv-platform-tags/src/tags.rs

```diff
@@ -912,6 +1047,7 @@ mod tests {
             Arch::X86_64,
         ))
         .unwrap();
+        let tags = tags.iter().map(ToString::to_string).collect::<Vec<_>>();
```

Comment:
No changes here which is really nice. Confirms that the tag generation and formatting remains unchanged.